components/crud-web-apps/jupyter/backend/apps/default/routes/post.py

```diff
@@ -27,6 +27,9 @@ def post_pvc(namespace):
 
     form.set_notebook_image(notebook, body, defaults)
     form.set_notebook_image_pull_policy(notebook, body, defaults)
+    form.set_notebook_http_rewrite_uri(notebook, body, defaults)
+    form.set_http_headers_request_set(notebook, body, defaults)
+    form.set_server_type(notebook, body, defaults)
```

Comment:
Could you also add this function to the Rok flavor of the app as well? In the `backend/apps/rok/routes/post.py` file